sipa/model/user.py

```diff
@@ -152,7 +152,12 @@ def traffic_history(self) -> list[dict]:
 
     def generate_rows(self, description_dict: dict) -> t.Iterator[Row]:
         for key, val in description_dict.items():
-            yield Row(description=val, property=getattr(self, key))
+            yield Row(property=getattr(self, key), **self.__text_to_dict(val))
+
+    def __text_to_dict(self, val: str|list[str]) -> dict:
+        if isinstance(val, list):
+            return {'description': val[0], 'subtext': val[1] if len(val) == 2 else val}
+        return {'description':val, 'subtext': None}
```

Comment:
Please do not use two underscores unless you know what they do! They have special behaviour (mangling).
I would call this `_parse_description` instead.
A `match` like this (warning: untested) might be easier to read:
```py
match val:
    case [d, s]:
        return {"description": d, "subtext": s}
    case [d]:
        return {"description": d, "subtext": None}
    return {"description": val, "subtext": None}

Reply:
should be resolved now